cristin-project/src/main/java/no/unit/nva/cristin/projects/NvaProjectBuilder.java

```diff
@@ -53,8 +59,8 @@ private static NvaContributor createNvaContributorFromCristinPersonByRole(Cristi
                                                                               CristinRole role) {
         NvaContributor nvaContributor = new NvaContributor();
         nvaContributor.setType(cristinRolesToNva.get(role.getRoleCode()));
-        nvaContributor.setIdentity(NvaPerson.fromCristinPerson(cristinPerson));
-        nvaContributor.setAffiliation(NvaOrganization.fromCristinInstitution(role.getInstitution()));
+        nvaContributor.setIdentity(Person.fromCristinPerson(cristinPerson));
```

Comment:
Maybe use cristinPerson.toPerson() instead, but that can be done later

Reply:
The NvaPerson is to be replaced with the new one with more properties